modules/data-entry/src/main/resources/SLING-INF/content/libs/cards/dataEntry/SelectableArea/Face.json

```
@@ -3,6 +3,7 @@
     "name": "Face",
     "image": "/libs/cards/resources/media/dataEntry/fullBody.svg",
     "jcr:path:map": "/libs/cards/resources/media/dataEntry/fullBodyMap.json",
+    "jcr:path:zones": "/libs/cards/resources/media/dataEntry/faceZones.json",
```

It was decided to list the default options directly in the variant definition as an unstructured child node named `defaultOptions`:

```suggestion
    "defaultOptions": {
      "jcr:primaryType" : "nt:undtructured",
      "face": "Face",
      "leftjaw": "Left Jaw",
      "rightjaw": "Right Jaw",
      "jaw": "Jaw",
      "throat": "Throat"
    },
```